protected/components/TargetCheckManager.php

```diff
@@ -61,10 +61,24 @@ public function create(Check $check, $data) {
                 $targetCheckScript->target_check_id = $targetCheck->id;
                 $targetCheckScript->save();
             }
+            if ($data["language_id"]){
+                $language = System::model()->findByPk($data["language_id"])->language;
```

Comment:
System always has a single row with id = 1, so it doesn't make sense to search using $data["language_id"] here.